testing/sdk_install.py

```diff
@@ -41,14 +41,15 @@ def install(
         timeout_sec=timeout_seconds,
         expected_running_tasks=running_task_count)
 
+    # Regardless of wait_scheduler_idle, it's safe to wait for the deployment from an install to complete
+    # before proceeding. This can take a while, default is 15 minutes. for example with HDFS, we can hit
+    # the expected total task count via FINISHED tasks, without actually completing deployment.
+    sdk_utils.out("Waiting for {}/{} to finish deployment plan...".format(package_name, service_name))
+    sdk_plan.wait_for_completed_deployment(service_name, timeout_seconds)
+
     # 2. Wait for the scheduler to be idle (as implied by deploy plan completion and suppressed bit)
     # This should be skipped ONLY when it's known that the scheduler will be stuck in an incomplete state.
     if wait_scheduler_idle:
```

Comment:
This flag now only controls whether we check for the `suppressed` endpoint. Should the parameter not be renamed? Also, is there a way to discover if the endpoint is present? We could check for a 404 and not wait on that?